.gitmodules

```diff
@@ -1,6 +1,12 @@
 [submodule "lib/PublishQueueAsyncRK"]
 	path = lib/PublishQueueAsyncRK
-	url = https://github.com/rickkas7/PublishQueueAsyncRK.git
+	url = git@github.com:rickkas7/PublishQueueAsyncRK.git
```

Comment:
Just a small thing, but since these are public repos, could you change these to point to the HTTPS clone URL? As it stands, the submodules will not clone from GitHub unless your SSH key is explicitly set.

It's not a huge issue for the most part, but it would save a troubleshooting step in case somebody in the future tries to clone without an SSH key registered to their account.

Reply:
Fixed!